virttest/qemu_monitor.py

```diff
@@ -906,11 +910,13 @@ def cmd(self, cmd, timeout=CMD_TIMEOUT, debug=True, fd=None):
             # Read any data that might be available
             self._recvall()
             if fd is not None:
-                if self._passfd is None:
-                    self._passfd = passfd_setup.import_passfd()
-                # If command includes a file descriptor, use passfd module
-                self._passfd.sendfd(self._socket, fd, b"%s\n" % cmd.encode())
-                self._log_lines(cmd)
+                if sys.version_info[:2] >= (3, 3):
```

Comment:
I think it'd make sense to judge this only once on import and define a `sendmsg` function that would either use py2 or py3 implementation, so here we'd only use it. What do you think? (the difference is quite big as here every `cmd` execution needs to judge the python version and we need to pay attention and keep adding the complex code to every place - nowadays 2 locations - so doing it once and simply calling our `_sendmsg` would be way easier. What do you think?

Reply:
yeah,can't agree more,thanks a lot for your suggest.